pkg/addonmanager/controllers/registration/controller.go

```diff
@@ -157,10 +157,17 @@ func (c *addonConfigurationController) sync(ctx context.Context, syncCtx factory
 		managedClusterAddonCopy.Status.Namespace = managedClusterAddonCopy.Spec.InstallNamespace
 	}
 
+	meta.SetStatusCondition(&managedClusterAddonCopy.Status.Conditions, metav1.Condition{
+		Type:    "RegistrationApplied",
+		Status:  metav1.ConditionTrue,
+		Reason:  "RegistrationApplied",
```

Comment:
use the type and reason defined in api repo

Reply:
fixed